README.md

```diff
@@ -2,7 +2,7 @@
 
 This document describes how the STAC (SpatioTemporal Asset Catalog) Project Steering Committee makes decisions on STAC and STAC API project issues. STAC is concerned with both the [stac-spec](https://github.com/radiantearth/stac-spec) and [stac-api-spec](https://github.com/radiantearth/stac-api-spec) repositories.
 
-In brief the committee votes on proposals on <https://github.com/radiantearth/stac-psc/issues>. Proposals are available for review for at least two business days, and a single veto is sufficient to delay progress though ultimately a majority of members can pass a proposal.
+In brief the committee votes on proposals on <https://github.com/radiantearth/stac-psc/issues>. Proposals are available for review for at least five business days, and a single veto is sufficient to delay progress though ultimately a majority of members can pass a proposal.
```

Comment:
We recently decided in openEO to change from two to five because two seem overly short and may not be enough to actually get to it. I think it's a good idea to extend this for STAC, too.